### DISCUSSION TITLE
Chia has become a 'Cult' - HDD Failure Sleep Depression? Is there a 12 step program for Chia-Addicts?

### DISCUSSION THREAD

**Discussion by room101-dev** [Chia-Network/chia-blockchain#5784] [General]
People should just admit a mistake and move on, the biggest regret I'm hearing is that they didn't make any money, but then why haven't you been on HPOOL all along, earning chia, waiting for what?

Then there is this drive loss problem, 100's of people bought these 18tb drives, filled them up, and they fail, then the people go nuts about losing months of plots.

Drives die, I just had a 2TB drop dead the other day, controller failure ordered a pcb from china, pulled the old prom-bios, and it still didn't come back to life. Shit happens, Chia-Plotting kill's my NVME's every 100 plots, again shit happens.

Just dial it back, maybe 4tb drives hold 38 plots, is less risky than a 18tb, holding 175, doesn't take as long to replace.
I just bought two 8tb's and one was dead, its taking seagate almost 3 weeks to get me the new one, seller didn't help me because I failed to report the first 5 days of purchase, I didn't think  to test, as the drives are piled up and I bring them in when I need new space.

Like everything only do what you can afford to lose, and mix&match, I was already mining gpu ETH done this for years, and I hack bitcoin as hobby, I'm only doing chia to diversify my crypto-hobby. A little space mining, a little gpu, and some classic btc coding.

CHIA is now a cult, and sadly too many people new to crypto have latched on to the cult bullshit about not using HPOOL, I knew from the very start that Solo-Mining was impossible, and I also knew that they were lying about chia, I knew the site was in english, I knew hpool was a trustful legitimate pool, all my pools are in china, so what?

What you have here is an over-hyped cult, and the next step is mass suicide, take a break folks, step back, quit waiting for the morons at chia-dev to develop a pool-protocol, mine what you have make some money and move on.

Quit putting all your eggs in one basket, I think 8tb drives are just right, the security kind we call WD purple, keep it simple; Take a rest from, and for gods sake quit waiting to make money, I think the biggest problem in CHIA is that people have spent months grinding their HDD's, and wasting their money, with NO ROI. For god's sake stop the insanity, the Chia-DEV are already terminal, you just watch their videos, and you can see the Cohen & Hoffman, are already exhausted emotionally & physically.

Everybody needs to take a break.

I make $20/day, on each batch of 200 plots, about the same as I would make on my ETH mining rigs;  I never expected anything more or less, Chia Cultists are holding out for the 'moon' but all they have gotten is the shaft. This is not going to end well for anybody.

**Comment by padik11** [comment]
You are very much right about what you write. An alternative to hard drives are SAS and Sata hard drives, but unfortunately on the chia website they recommended USB.

What pool are you using?
